### ISSUE TITLE
Redesigning Luanti Banner

### ISSUE THREAD

**Issue by SaiPanneerselvam** [luanti-org/luanti#15590]
In #15428, @SmallJoker suggested that a discussion (in sense) be made regarding the topic of Luanti's Menu Banner and the idea of replacing it. People can post their new logo ideas and designs until one is picked (make sure it is CC-BY-SA or relevant). Thanks!

**Comment by SwissalpS** [comment]
duplicate of https://github.com/minetest/minetest/issues/15428 (by same user)

**Comment by Zughy** [comment]
Worth noting from https://github.com/minetest/minetest/issues/6733#issuecomment-2480880150 :

![image](https://github.com/user-attachments/assets/63ba6160-1046-475f-8b74-79a0f70db3f3)

**Comment by nininik0** [comment]
I'm working on drawing one with the main menu colors

**Comment by siliconsniffer** [comment]
> Worth noting from [#6733 (comment)](https://github.com/minetest/minetest/issues/6733#issuecomment-2480880150) :
> 
> ![image](https://private-user-images.githubusercontent.com/63455151/398299005-63ba6160-1046-475f-8b74-79a0f70db3f3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUxMjA5MzUsIm5iZiI6MTczNTEyMDYzNSwicGF0aCI6Ii82MzQ1NTE1MS8zOTgyOTkwMDUtNjNiYTYxNjAtMTA0Ni00NzVmLThiNzQtNzlhMGY3MGRiM2YzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI1VDA5NTcxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUyNzNlYzlkOTk2NGVjZDcxNWFiNTZhYzU2YTI3NTA2YmEyMThjYzFhYTU5YmJiYTJiY2ZjYjhmNzQ4OTBlNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qONw9tOuWIuKcGCGKMIl90Rrm6x2TA0UBMAcEoMCrGs)

![image](https://github.com/user-attachments/assets/6ca66c69-8676-4b8e-923d-dbb079d0efa9)
I quite like it but I am uncertain if it fits the rest. For me, it looks a bit "ahead of its time", 
Personally, I'd still prefer this over the current one.

**Comment by SaiPanneerselvam** [comment]
This is the main menu logo made by WSOR on github (check #15428 and [here](https://discord.com/channels/369122544273588224/369123125021114369/1307419132211368027) for more info)'
Shown in the Luanti main menu.
![image](https://github.com/user-attachments/assets/7c566810-7fa0-4ae5-8f1f-1ddf0bb19fde)
ME: This logo kind of looks small to the interface, quick fix would be sizing the image larger, but will do that on request

##### This looks like a slightly modified version of @Zughy 's icon.

**Comment by appgurueu** [comment]
Are you sure that this is by wsor / Zughy? The original artwork here is from @Sumianvoice as far as I remember, Zughy and wsor have just tweaked it.

**Comment by SaiPanneerselvam** [comment]
> Are you sure that this is by wsor / Zughy? The original artwork here is from @SumianVoice as far as I remember, Zughy and wsor have just tweaked it.

Good to know

**Comment by Wuzzy2** [comment]
I've already wrote my feedback on the "blocky" text logo in #15428 but I repeat the short version here:

I like the style and idea very much. I think the "blocky" style, along with the abstract colors, just fit Luanti perfectly.

My biggest complaint is the letter "N" because it doesn't really look like an N anymore. And maybe the logo might look better in orthogonal projection. But that's about it. For details, just go back to my post in that other issue (but it's not that important).

**Comment by siliconsniffer** [comment]
Zughy also made a "pixelified" version which I quite like. Fits with the style of the icons.
![image](https://github.com/user-attachments/assets/6a39161c-8841-48aa-8424-0411d6b208f7)
(Window was resized, hence the image is a bit smaller)

**Comment by siliconsniffer** [comment]
> > I quite like it but I am uncertain if it fits the rest. For me, it looks a bit "ahead of its time"
> 
> What do you mean by that? Like "LUANTI IN 2050" or something?

What I mean by that is that I think that the style of the pixelated image doesn't harmonize well with the style of the current GUI, which makes it "stick out" style-wise from the rest.

On the other hand, it does fit nicely with the icons. As already stated, I'd prefer having this over the current one.

**Comment by SaiPanneerselvam** [comment]
> > > I quite like it but I am uncertain if it fits the rest. For me, it looks a bit "ahead of its time"
> > 
> > What do you mean by that? Like "LUANTI IN 2050" or something?
> 
> What I mean by that is that I think that the style of the pixelated image doesn't harmonize well with the style of the current GUI, which makes it "stick out" style-wise from the rest.
> 
> On the other hand, it does fit nicely with the icons. As already stated, I'd prefer having this over the current one.

To be honest, this logo seems like has a "multicrap" kind of vibe

Edit, by multicrap I only meant multicraft

**Comment by Wuzzy2** [comment]
#16849 doesn't even mention the Luanti text logo, so I don't see how it justifies closing as duplicate.

Please reopen. I think there should be one dedicated issue only for the Luanti text logo (i.e. this issue).

**Comment by siliconsniffer** [comment]
I agree. As long as there is no new logotype/banner, there is no reason to close this.

My personal opinion hasn't really changed, but I am unsure how to move forward with my PR. Judging by the reactions, opinions on the proposed logotype are split 50/50. The lack of alternative proposals doesn't help either.

Ultimately, a new banner should appeal to new as well as current players while staying somewhat neutral design-wise.

**Comment by SaiPanneerselvam** [comment]
I had closed this while meaning to mark it as stale as though the issue is still prominent, this specific issue is 1year old at this point. It is alright if this issue will stay open, as I was just thinking about it to clear my inbox a bit

Also:
Does anyone plan on upgrading the interface too, maybe to match the logo that was suggested earlier in this issue?

**Comment by CrazyladMT** [comment]
> Does anyone plan on upgrading the interface too

#6733
